Transfer/Sources/TransferNotesCell.swift

```diff
@@ -36,7 +36,7 @@ final class TransferNotesCell: UITableViewCell {
         return textField
     }()
 
-    private var isHideBorder: Bool = false
+    private var hideBorder: Bool = false
```

Comment:
Just curious if the naming convention is top use `is` or not?

Reply:
Naming convention is to use `is` but here it should either be shouldHideBorder or hideBorder because this var decides whether to hide border border or not.

Reply:
or it should be isHiddenBorder